packages/web/src/components/track/PremiumContentLabel.tsx

```diff
@@ -29,21 +29,7 @@ export const PremiumContentLabel = ({
     return null
   }
 
-  if (isOwner) {
```

Comment:
Are we no longer showing the labels if `isOwner` or do they appear somewhere else?

Reply:
we are. the logic is refactored and that path would hit the below block